timer.r

```diff
@@ -0,0 +1,14 @@
+# Load the tictoc package
+library(tictoc)
+
+# List all the R script files in the "scripts" directory
+script_files <- list.files(path = "scripts/", pattern = "\\.R$", full.names = TRUE)
```

Comment:
This does not work when I run it. The issue seems to be that you use an uppercase `.R` when the files have lowercase `.r` file extensions.